style.css

```diff
@@ -0,0 +1,206 @@
+*{
+    margin:0;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+.grid{
+    display: grid;
+    grid-template: "left main right";
+    grid-template-columns: 1fr 800px 1fr ;
+    
+}
+
+.header{
+    display:grid;
+    padding-top: 15px;
+    font-weight: 500;
+    grid-template-rows: repeat(auto-fill);
+}
+.header > hr{
```

Comment:
I didnt know we could style <hr>, thank you for this info thomas:)

Reply:
Yeah, i never thought of this too, great stuff! xD